### PR TITLE
Add support to spoof releases

### PR DESCRIPTION
**Pull request by styxit**

### CODE REVIEW

File: spoof [hunk 1]
```diff
@@ -4,15 +4,18 @@ require __DIR__.'/vendor/autoload.php';
...
 
-(new Application('spoof', '0.0.1'))
-    ->add(new PrMergeCommand())
-    ->getApplication()
-    ->setDefaultCommand('PrMerge', true)
-    ->run();
+
+$application = new Application('spoof', '0.0.1');
+
+$application ->add(new PrMergeCommand());
```
Comment (robbinjanssen) [inline spoof]: Remove space

File: src/Commands/PublishReleaseCommand.php [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                'tag',
+                InputArgument::REQUIRED,
+                'The tag to fake. Example: v1.0.2'
+            );
+    }
+
+    /**
+     * Execute the command.
+     *
+     * @param InputInterface  $input
```
Comment (robbinjanssen) [inline src/Commands/PublishReleaseCommand.php]: Missing description

File: src/Commands/PublishReleaseCommand.php [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+            ]
+        );
+
+        $output->writeLn('Done.');
+    }
+
+    /**
+     * Ask a question the user must answer with 'y' or 'n'.
+     *
+     * @param InputInterface  $input
```
Comment (robbinjanssen) [inline src/Commands/PublishReleaseCommand.php]: Missing description

File: src/Commands/PublishReleaseCommand.php [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+     * @param OutputInterface $output
+     * @param string          $question The question to ask.
+     *
+     * @return bool True when the user entered 'y', False otherwise.
+     */
+    private function confirm(InputInterface $input, OutputInterface $output, $question = 'Ok?')
+    {
+        $helper = $this->getHelper('question');
+        $confirmQuestion = new ConfirmationQuestion($question.' [y/n]: ', false);
```
Comment (trizz) [inline src/Commands/PublishReleaseCommand.php]: Perhaps `[y/N]` to indicate the `No` is the default.

**Review by robbinjanssen — CHANGES_REQUESTED** [review]

**Review by trizz — APPROVED** [review]

**Pull request merged**